{
    "type": "bundle",
    "id": "bundle--ccea812d-a9e1-4fb6-8db2-46ca1d840b11",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--df817061-beb5-4320-8815-d6f861442705",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2025-04-03T00:39:50.470805Z",
            "modified": "2025-04-03T00:39:50.470805Z",
            "name": "CVE-2025-21975",
            "description": "In the Linux kernel, the following vulnerability has been resolved:\n\nnet/mlx5: handle errors in mlx5_chains_create_table()\n\nIn mlx5_chains_create_table(), the return value ofÂ mlx5_get_fdb_sub_ns()\nand mlx5_get_flow_namespace() must be checked to prevent NULL pointer\ndereferences. If either function fails, the function should log error\nmessage with mlx5_core_warn() and return error pointer.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2025-21975"
                }
            ]
        }
    ]
}